eval/src/vespa/eval/tensor/dense/dense_tensor_builder.h

```diff
@@ -28,7 +29,8 @@ class DenseTensorBuilder
     size_t calculateCellAddress();
 
 public:
-    DenseTensorBuilder();
+    DenseTensorBuilder() : DenseTensorBuilder(eval::ValueType::CellType::DOUBLE) { }
+    DenseTensorBuilder(eval::ValueType::CellType cellType);
```

Comment:
enabling specifying that cells are float when they are in fact double may be confusing

Reply:
Reverted as discussed